plotly/tools.py

```diff
@@ -307,33 +309,31 @@ def get_embed(file_owner_or_url, file_id=None, width="100%", height=525):
         raise exceptions.PlotlyError(
             "The 'file_id' argument must be a non-negative number."
         )
-    if isinstance(width, int):
+    if share_key is None:
         s = ("<iframe id=\"igraph\" scrolling=\"no\" style=\"border:none;\""
              "seamless=\"seamless\" "
              "src=\"{plotly_rest_url}/"
-             "~{file_owner}/{file_id}.embed"
-             "?width={plot_width}&height={plot_height}\" "
+             "~{file_owner}/{file_id}.embed\" "
              "height=\"{iframe_height}\" width=\"{iframe_width}\">"
              "</iframe>").format(
             plotly_rest_url=plotly_rest_url,
             file_owner=file_owner, file_id=file_id,
-            plot_width=width - padding, plot_height=height - padding,
             iframe_height=height, iframe_width=width)
     else:
         s = ("<iframe id=\"igraph\" scrolling=\"no\" style=\"border:none;\""
              "seamless=\"seamless\" "
              "src=\"{plotly_rest_url}/"
-             "~{file_owner}/{file_id}.embed\" "
+             "~{file_owner}/{file_id}.embed?{share_key}\" "
```

Comment:
oh man, that's super confusing (if i'm understanding correctly). so as a user, do i need to know to pass in `share_key='share_key=ASDFIH24HS3'`?

Ideally (imo), our api here is `append_share_key=True` or something and we just handle it.

At least could it be `share_key='ASDFIH24HS3'`?

Also, whatever we need to go with, let's give an example in the docstring for it?